File: comms/push-notifications/apps/update-app/update-app.cs
using Twilio.Comms;
using System.Threading.Tasks;
using Twilio.Comms.PushNotifications;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioComms(
            accountId: "<username>",
            authToken: "<password>"
        );

        await client.PushNotifications.Apps.UpdateAsync(
            "appName",
            new AppsUpdateRequest {
                IsDefault = true
            }
        );
    }

}

File: comms/push-notifications/apps/update-app/update-app.php
<?php

namespace Example;

use Twilio\Comms\TwilioComms;
use Twilio\Comms\PushNotifications\Apps\Requests\AppsUpdateRequest;

$client = new TwilioComms(
    accountId: '<username>',
    authToken: '<password>',
);
$client->pushNotifications->apps->update(
    'appName',
    new AppsUpdateRequest([
        'isDefault' => true,
    ]),
);
